release-notes/DxMs/DataAggregator_change_log.md

```diff
@@ -4,10 +4,57 @@ uid: DataAggregator_change_log
+
+Data Aggregator Installer has now been upgraded to use WiX version 4.0.4.
+
 #### 13 February 2024 - Enhancement - DataAggregator 3.0.2 - Debug UI shows job names instead of job IDs [ID_38697]
 
 In the debug UI of Data Aggregator (accessible via `http://<hostname or IP>:<Data Aggregator port>/debug/`) the different tab pages will now show the job name instead of the job ID, so that it is easier to find a specific job.
 
+#### 07 February 2024 - Enhancement - DataAggregator 3.0.1 - Include bs4 Python library with DataAggregator installer [ID_38654]
+
+The installer has been extended to include the bs4 pip package (and its dependencies) by default in the DataAggregator installer.
+
+#### 07 February 2024 - Enhancement - DataAggregator 3.0.1 - Improve DataAggregator Migrator [ID_38682]
+
+Improvements have been made to the [migrator tool](xref:Data_Aggregator_Migrator) to better allow converting GQI queries from the [previous syntax to the new one](xref:Data_Aggregator_settings#gqi-queries).
+
+This includes:
+- Extra information added to the Help command (_-h_ or _--help_)
+- Change how the [DataAPI soft launch](xref:Overview_of_Soft_Launch_Options#dataapi) verification is applied in order to facilitate the migration process for users yet using it.
+
+#### 31 January 2024 - New Feature - DataAggregator 3.0.0 - Support for Data Sources and integration with DataAPI [ID_38307] [ID_38404] [ID_38234] [ID_38496] [ID_38560]
+
+Data Aggregator has been extended with support for [Data Sources](xref:Data_Sources) and Scripted connectors.
+This module offers an easy solution to access data from diverse sources and swiftly integrate new products with DataMiner.
+
+This feature is behind the [DataAPI soft launch](xref:Overview_of_Soft_Launch_Options#dataapi) option and will display an alphabetically sorted list of your Data Sources.
+
+###### Include Python environment with Data Aggregator installer [ID_38064]
+
+Python environment and libraries come with Data Aggregator's installer so no need to configure that after installation.
+Making it possible to start using the feature out of the box.
+
+In addition, several Python libraries come pre-installed but more can be installed following [these steps](xref:Data_Sources_Setup#installing-extra-python-packages).
+
+##### Enforce authentication when accessing Data Aggregator DxM web interface via reverse proxy [ID_38275]
+
+To ensure better security, access to Data Aggregator's web interface now requires the user to authenticate itself.
+
+##### API Gateway module registration [ID_38570]
+
+Data Aggregator will now register itself with API Gateway allowing for an overview of all node instances.
+
+##### Extend Data Aggregator to have a CRUD API for Data Sources [ID_37309]
+
+Data Aggregator API has been extended from the ability control the Jobs flow (start, stop and check status) to a full CRUD.
+This has been done for both Jobs and Data Sources, allowing for configuration with less manual steps.
```

Comment:
I find this description a little confusing. Could you rephrase this keeping the end user in mind? E.g. what is "control the Jobs flow to a full CRUD"? I'm also confused by the first sentence mentioning "the Jobs flow" and the second one saying that this "has been done for both Jobs and Data Sources".

Reply:
Slight rephrasing